pkg/controller/nodeipam/ipam/cidr_allocator.go

```diff
@@ -70,11 +70,11 @@ const (
 	// cidrUpdateRetries is the no. of times a NodeSpec update will be retried before dropping it.
 	cidrUpdateRetries = 3
 
-	// updateRetryTimeout is the time to wait before requeing a failed node for retry
-	updateRetryTimeout = 250 * time.Millisecond
+	// updateRetryTimeoutMilliseconds is the time to wait before requeing a failed node for retry
+	updateRetryTimeoutMilliseconds = 250
```

Comment:
Why this change - I think having it as a duration is more intuitive.

Reply:
Done.